pkgs/by-name/ir/ironbar/package.nix

```diff
@@ -44,6 +47,9 @@ rustPlatform.buildRustPackage rec {
     gsettings-desktop-schemas
     libxkbcommon
     openssl
+    luajit
+    lua51Packages.lgi
+    libpulseaudio
```

Comment:
These packages are only required when the features are enabled - they are by default, so not a biggie, but it's possible to slim down a bit:

https://github.com/JakeStanger/ironbar/blob/d37547e046b50c418b6e388a249fd3e6aedfdd75/nix/default.nix#L57-L59

Reply:
I don't think this package even feature checks

Reply:
This package doesn't currently feature check, looking at the file provided upstream there's a decent discrepancy between the two. 

 In https://github.com/NixOS/nixpkgs/pull/307726/commits/9ff1da6d1de964ce27e11f944b68e36e0097805b I've added a features attribute as well as the hasFeature function which can allow `features` to be overridden.